### PR TITLE
feature/list-of-bought-rewards-in-admin-panel

### PR DESCRIPTION
**Pull request by rniedziolka**
Hi,
next feature ready:
https://www.loom.com/share/5fbd7c3c1d1840fcb40c8b64849f3fae
Waiting for the feedback!

### CODE REVIEW

File: app/controllers/admin/orders_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Admin
+  class OrdersController < AdminController
+    def index
+      @orders = Order.all
```
Comment (ciejj) [inline app/controllers/admin/orders_controller.rb]: This is en example of n+1 problem :)

**Review by ciejj — APPROVED** [review]
Nice! but let's try to fix that n+1 problem :)

File: app/controllers/admin/orders_controller.rb (hunk 1 above)
Comment (rniedziolka) [reply app/controllers/admin/orders_controller.rb]: Fixed.

**Comment by rniedziolka** [comment]
n+1 problem fixed here: https://github.com/rniedziolka/employee-recognition-platform-1637255604/pull/22/commits/685d7b8c8f5236973bf43d716f01b02b341b8692

**Review by oskarlakner — APPROVED** [review]

**Pull request merged**